updated Dashboard, Pivot EUD 20.1 (#107)
DevExpress/dotnet-eud: interface-elements-for-web/articles/pivot-table/data-presentation/filter-data/build-complex-filter-criteria.md

legacyId: 11063
---
# Build Complex Filter Criteria
Use a **Filter Editor** to build complex filter criteria with an unlimited number of filter conditions, combined by logical operators.

## Filter Editor

> To invoke the Filter Editor, click an empty space within the Pivot Table header region and select **Show Filter** from the context menu.

Reviewer: ```suggestion
To invoke the Filter Editor, click an empty space within a Pivot Table's header region and select **Show Filter** from the context menu.
```

![EU_Filter](../../../../images/img15870.png)

> The Pivot Table synchronizes filters applied in the [Filter Windows](filter-data-by-field-values/using-filter-popup-windows.md) and Filter Editor. You can customize the resulting filter sting or clear it to reset the applied filters.

Reviewer: ```suggestion
The Pivot Table synchronizes filters applied in [Filter Windows](filter-data-by-field-values/using-filter-popup-windows.md) and the Filter Editor. You can customize the filter sting or clear it to reset applied filters.
```

> If you use a **classic** mode, a **Prefilter** is available instead.

Reviewer: ```suggestion
If you use **classic** mode, a **Prefilter** is available instead.
```

> The Prefilter does not affect filters applied in the Filter Windows.

Reviewer: ```suggestion
Note that the Prefilter does not affect filters applied in Filter Windows.
```

To invoke the Prefilter, click an empty space within the Pivot Table header region and select **Show Prefilter** from the context menu.

![EU_ShowPrefilter](../../../../images/img15872.png)

## Filter Panel

> After you built and applied a filter condition, a **Filter Panel** appears at the Pivot Table bottom. This panel displays the filter condition.

Reviewer: ```suggestion
After you have built and applied a filter condition, a **Filter Panel** appears at the bottom of the Pivot Table. This panel displays the filter condition.
```

![EU_PrefilterPanel](../../../../images/img15871.png)

> The Filter Panel displays a filter expression, a check box used to temporarily enable/disable the filtering, and a clear button used to reset the current filter condition.

Reviewer: ```suggestion
The Filter Panel displays a filter expression, a check box used to temporarily enable/disable the filter, and a 'clear' button used to reset the current filter condition.
```

> To modify a filter condition, open the Filter Editor. Click a filter condition displayed in the filter panel to invoke it.

Reviewer: ```suggestion
To modify a filter condition, open the Filter Editor. In the filter panel, click a filter condition to invoke it.
```

> For information on the Filter Editor filtering capabilities, see [Filter Editor](../../../filter-editor.md).

Reviewer: ```suggestion
For information on the Filter Editor's filtering capabilities, see the following help topic: [Filter Editor](../../../filter-editor.md).
```
